### PR TITLE
[pentair] Some improvements to the pentair documentation.

### PR DESCRIPTION
**Pull request by mcarbs**
Fixed syntax error in the example items, added USB Serial example and expanded the sitemap to be a complete example.

Signed-off-by: Mark Carbonaro <mark.carbonaro@gmail.com>

### CODE REVIEW

File: bundles/org.openhab.binding.pentair/README.md [hunk 1]
```diff
@@ -136,66 +145,71 @@ Pentair things support a variety of channels as seen below in the following tabl
...
+Number Pump_Mode               "Pump Mode [%d]"                               (gPool) { channel = "pentair:intelliflo:1:pump1:mode" }
+Number Pump_RPM                "Pump RPM [%d]"                                (gPool) { channel = "pentair:intelliflo:1:pump1:rpm" }
+Number Pump_Power              "Pump Power [%d W]"                            (gPool) { channel = "pentair:intelliflo:1:pump1:power" }
+Number Pump_Error              "Pump Error [%d]"                              (gPool) { channel = "pentair:intelliflo:1:pump1:error" }
+Number Pump_PPC                "Pump PPC [%d]"                                (gPool) { channel = "pentair:intelliflo:1:pump1:ppc" }
 ```
 
-Here is an example sitemap:
-
+Here is an example of a complete sitemap, saved as pentair.sitemap.  Adjust the temperature values for metric is so desired.
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.pentair/README.md:189]: Can you put each sentence into a separate line and maybe use backticks to surround the file name here and the others above?

```suggestion
Here is an example of a complete sitemap, saved as `pentair.sitemap`.
Adjust the temperature values for metric is so desired.
```

**Comment by jsjames** [comment]
I have reviewed the suggested edits and these updates can be merged.

File: bundles/org.openhab.binding.pentair/README.md (hunk 1 above)
Comment (cpmeister) [reply bundles/org.openhab.binding.pentair/README.md:189]: Also just to add on to this.
```suggestion
Here is an example of a complete sitemap, saved as `pentair.sitemap`.
Adjust the temperature values to metric if so desired.
```

**Comment by fwolter** [comment]
@mcarbs Do you accept the requested changes? Can you commit them to this PR?

**Comment by mcarbs** [comment]
Sorry I missed this, I have made the suggested updates

**Comment by fwolter** [comment]
Can you fix the DCO check? Seems like you used another mail address.

**Comment by mcarbs** [comment]
I've messed this up, i'll open up another PR

**Comment by fwolter** [comment]
No need to, please stick to this PR. The DCO check can be fixed.

**Comment by fwolter** [comment]
Clicking on the DCO check gives you the Git commands to fix it.

**Comment by fwolter** [comment]
Do you need any help?

**Comment by jsjames** [comment]
This is approved by me.

**Review by Hilbrand — APPROVED** [review]
LGTM; comment by other reviewers is addressed

**Pull request merged**